### ISSUE TITLE
Stats store info with nickname instead of user id

### ISSUE THREAD

**Issue by asteriskman7** [r-incremental-games/TheCount#2]
Currently, stats are saved vs nickname. This breaks when users change their nickname.

This needs to change to saving stats via user id. When stats are displayed convert the user id to nickname. May need additional code to deal with user ids that are no longer in the server.

Should include code to do a 1 time conversion of old stats to new format.

**Comment by Ishadijcks** [comment]
Maybe whenever a user with such an existing name sends a message, it moves the stat to that id and deletes the name record?

Or a one-off command that checks everyone on the server (if that's possible)

**Comment by asteriskman7** [comment]
Yeah, whatever reasonable logic to get from where we are to where we should be is ok. This is just for fun so it doesn't need to be perfect.
